components/org.wso2.carbon.identity.api.user.authorized.apps/org.wso2.carbon.identity.rest.api.user.authorized.apps.v2/src/main/java/org/wso2/carbon/identity/rest/api/user/authorized/apps/v2/core/Constants.java

```diff
@@ -49,7 +49,11 @@
                         "user: %s"),
         ERROR_CODE_GET_APP_BY_USER("15005", "Error retrieving authorized applications",
                 "A system error occurred while retrieving authorized applications for " +
-                        "user: %s");
+                        "user: %s"),
+        ERROR_CODE_REVOKE_TOKEN_BY_APP_ID("10006", "Error revoking issued tokens", "A system " +
```

Comment:
for system errors the error code should be something like `15006`

Reply:
Addressed in [81a8d70](https://github.com/wso2/identity-api-user/pull/129/commits/81a8d7074753b74f8bb43429db8d4ff80383675f)